src/builtin.d

```diff
@@ -69,7 +69,7 @@ extern (C++) Expression eval_sqrt(Loc loc, FuncDeclaration fd, Expressions* argu
 {
     Expression arg0 = (*arguments)[0];
     assert(arg0.op == TOKfloat64);
-    return new RealExp(loc, Port.sqrt(arg0.toReal()), arg0.type);
+    return new RealExp(loc, ddmd.root.real_t.sqrt(arg0.toReal()), arg0.type);
```

Comment:
This, and other functions in ``Port`` should remain there, as they have nothing to do with ``real_t`` per say, other than they accept and return that type.